ts/Stock/Indicators/Klinger/KlingerOptions.d.ts

```diff
@@ -0,0 +1,39 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import CSSObject from '../../../Core/Renderer/CSSObject';
+import type {
+    SMAOptions,
+    SMAParamsOptions
+} from '../SMA/SMAOptions';
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+export interface KlingerOptions extends SMAOptions {
+    params?: KlingerParamsOptions;
+    signal?: Record<string, CSSObject>;
```

Comment:
Can we use the same name as for MACD signal line? See: https://api.highcharts.com/highstock/series.macd.signalLine